lightning/src/ln/channelmanager.rs

```diff
@@ -2440,6 +2441,9 @@ where
 	/// See `PendingOutboundPayment` documentation for more info.
 	///
 	/// See `ChannelManager` struct-level documentation for lock order requirements.
+	#[cfg(test)]
+	pub(super) pending_outbound_payments: OutboundPayments,
+	#[cfg(not(test))]
 	pending_outbound_payments: OutboundPayments,
```

Comment:
nit: Would prefer to add a test-only method to look-up a payment by id instead of having conditional visibility.